### PR TITLE
Update for Cloudera Director 2.2 and Cloudera Director Azure Plugin 1.1

### PR DESCRIPTION
**Pull request by jasonbw**
This updates the Azure Plugin configs to for Cloudera Director 2.2 and Cloudera Director Azure Plugin 1.1

@mdzz - to review

### CODE REVIEW

File: configs/azure.reference.conf [hunk 1]
```diff
@@ -920,7 +952,7 @@ cluster {
     # Optional override of CDH parcel repositories
     #
 
-    # parcelRepositories: ["http://archive.cloudera.com/cdh5/parcels/5.9/"]
+    # parcelRepositories: ["http://archive.cloudera.com/cdh5/parcels/5.5/"]
```
Comment (mdzz) [inline configs/azure.reference.conf]: @jasonbw I think this needs to be reverted. Same comment for the other config samples.

**Comment by jasonbw** [comment]
@mdzz I reverted the change for `azure.reference.conf` and `azure.kerberos.conf`. I had to apply the change for `azure.simple.conf` as it had `5.5`

**Comment by jasonbw** [comment]
Offline +1 from @mdzz 

@jadair-cloudera - for merge

**Comment by jasonbw** [comment]
for merge - @jadair-cloudera, @bhavanki, @meowfaceman, @handavid, @andreisavu

File: configs/azure.reference.conf (hunk 1 above)
Comment (mdwn) [reply configs/azure.reference.conf]: +1. I imagine we want most of our reference files pointing to 5.9, not 5.5

**Comment by mdwn** [comment]
I'm +1 with this once azure.kerberos.conf and azure.reference.conf no longer point to 5.5 parcel repositories. I'll get one other person to look over this, then I can merge.

File: configs/azure.reference.conf (hunk 1 above)
Comment (jasonbw) [reply configs/azure.reference.conf]: I thought I had already fixed this, will fix it now

**Comment by jasonbw** [comment]
@meowfaceman - all azure.*.conf `parcelRepositories` should now point to `5.9`

**Pull request merged**